core/arch/arm/plat-poplar/platform_config.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+
+/* PL011 UART */
+#define CONSOLE_UART_BASE	PL011_UART0_BASE
```

Comment:
A define from `hi3798cv200.h` is used here, please include the file too.

Reply:
Ah, it was in `main.c`. Will remove it there and include it here instead. Thanks for spotting!